modules/pipeline/spec/pipeline_task.go

```diff
@@ -95,6 +95,8 @@ type PipelineTaskExtra struct {
 	LoopOptions *apistructs.PipelineTaskLoopOptions `json:"loopOptions,omitempty"` // 开始执行后保证不为空
 
 	AppliedResources apistructs.PipelineAppliedResources `json:"appliedResources,omitempty"`
+
+	EncryptSecretKeys []string `json:"encryptSecretKeys"`
```

Comment:
no need. pipeline snapshot level is enough now.

Reply:
the func `pullBootstrapInfo ` will use